src/visitors/templateLiterals/index.js

```diff
@@ -14,4 +16,9 @@ export default (path, state) => {
   } else if (useTranspileTemplateLiterals(state)) {
     transpile(path, state)
   }
+
+  // add uglify pure (/*#__PURE__*/) comment before function calls
+  if(useUglifyPure(state)){
```

Comment:
im not sure when but sometimes `path.node.leadingComments` might be null, it would probably be good to ensure the node has this as array before calling `annotateAsPure`, simple:
```js
path.node.leadingComments = path.node.leadingComments || []
annotateAsPure(path)

```
should do the trick